TLM/TLM/U/HelperExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace TrafficManager.U {
+    using ColossalFramework.UI;
+    using UnityEngine;
+
+    public static class HelperExtensions {
+        /// <summary>
+        /// Get rectangle of a control in GUI screen space.
+        /// GUI screen space is used to draw overlay textures in overlay helpers.
+        /// </summary>
+        public static Rect GetScreenRectInGuiSpace(this UIComponent ctrl) {
+            Vector3 absPosition = ctrl.absolutePosition;
+            float scaleX = Screen.width / 1920f;
+            float scaleY = Screen.height / 1080f;
```

Comment:
does this work if screen aspect ratio changes?

Reply:
GUI canvas seems to always maintain1920x1080 screen ratio internally. If this is not true, i need more info.